### ISSUE TITLE
Autodoc libraries missing the useful ones

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/lcls-plc-tmo-motion#46]
Just has the built-ins that are set to placeholders
![image](https://user-images.githubusercontent.com/10647860/105425092-8ddf3080-5bfd-11eb-965b-03d6610b138e.png)

**Comment by klauer** [comment]
Hmm, I'd guess this is meant to be an ads-deploy issue? https://github.com/pcdshub/ads-deploy/blob/master/ads_deploy/templates/%7B%7Btsproj.name%7D%7D_%7B%7Bplc.name%7D%7D_summary.rst

Wonder how this is differently structured than whatever project I was using when writing that template.

**Comment by ZLLentz** [comment]
Yeah I figured I would just report these on each repo that had something weird going on here- probably a systematic incompatibility between certain project structures and ads-deploy
